com.unity.mobile.notifications/Runtime/Android/Plugins/mobilenotifications.androidlib/build.gradle

```diff
@@ -12,6 +12,7 @@ def getCompileSdk(unityLib) {
 }
 
 android {
+    namespace "com.unity.androidnotifications"
```

Comment:
I see that 21.3 is failing due to this. 
Looks like this is a way to fix it: https://stackoverflow.com/questions/76108428/how-do-i-fix-namespace-not-specified-error-in-android-studio